frontend/config/plugins.js

```diff
@@ -15,8 +15,10 @@ const plugins = [
       './Navigation': resolve(__dirname, '../src/Navigation'),
     },
     shared: [
-      { 'react-router-dom': { requiredVersion: package.dependencies['react-router-dom'] } },
+      { 'react-router-dom': { singleton: true, import: false, requiredVersion: package.dependencies['react-router-dom'] } },
```

Comment:
These `requiredVersion: package.dependencies['module-name']` bits (here and below) might not be necessary.

AFAIK webpack module federation plugin code already attempts to auto-detect this information from dependencies in the project's `package.json` file.